pkg/eigenState/operatorAVSSplits/operatorAVSSplits.go

```diff
@@ -0,0 +1,279 @@
+	Network          config.Network
+	Environment      config.Environment
+	logger           *zap.Logger
+	globalConfig     *config.Config
+
+	// Accumulates state changes for SlotIds, grouped by block number
+	stateAccumulator map[uint64]map[types.SlotID]*OperatorAVSSplit
+}
+
+func NewOperatorAVSSplitModel(
+	esm *stateManager.EigenStateManager,
+	grm *gorm.DB,
+	logger *zap.Logger,
+	globalConfig *config.Config,
+) (*OperatorAVSSplitModel, error) {
+	model := &OperatorAVSSplitModel{
+		BaseEigenState: base.BaseEigenState{
+			Logger: logger,
+		},
+		DB:               grm,
+		logger:           logger,
+		globalConfig:     globalConfig,
+		stateAccumulator: make(map[uint64]map[types.SlotID]*OperatorAVSSplit),
+	}
+
+	esm.RegisterState(model, 8)
+	return model, nil
+}
+
+func (oas *OperatorAVSSplitModel) GetModelName() string {
+	return "OperatorAVSSplitModel"
+}
+
+type operatorAVSSplitOutputData struct {
+	ActivatedAt             uint64 `json:"activatedAt"`
+	OldOperatorAVSSplitBips uint64 `json:"oldOperatorAVSSplitBips"`
+	NewOperatorAVSSplitBips uint64 `json:"newOperatorAVSSplitBips"`
+}
+
+func parseOperatorAVSSplitOutputData(outputDataStr string) (*operatorAVSSplitOutputData, error) {
+	outputData := &operatorAVSSplitOutputData{}
+	decoder := json.NewDecoder(strings.NewReader(outputDataStr))
+	decoder.UseNumber()
+
+	err := decoder.Decode(&outputData)
+	if err != nil {
+		return nil, err
+	}
+
+	return outputData, err
+}
+
+func (oas *OperatorAVSSplitModel) handleOperatorAVSSplitBipsSetEvent(log *storage.TransactionLog) (*OperatorAVSSplit, error) {
+	arguments, err := oas.ParseLogArguments(log)
+	if err != nil {
+		return nil, err
+	}
+
+	outputData, err := parseOperatorAVSSplitOutputData(log.OutputData)
+	if err != nil {
+		return nil, err
+	}
+
+	activatedAt := time.Unix(int64(outputData.ActivatedAt), 0)
+
+	split := &OperatorAVSSplit{
+		Operator:                strings.ToLower(arguments[1].Value.(string)),
+		Avs:                     strings.ToLower(arguments[2].Value.(string)),
+		ActivatedAt:             &activatedAt,
+		OldOperatorAVSSplitBips: outputData.OldOperatorAVSSplitBips,
+		NewOperatorAVSSplitBips: outputData.NewOperatorAVSSplitBips,
+		BlockNumber:             log.BlockNumber,
+		TransactionHash:         log.TransactionHash,
+		LogIndex:                log.LogIndex,
+	}
+
+	return split, nil
+}
+
+func (oas *OperatorAVSSplitModel) GetStateTransitions() (types.StateTransitions[*OperatorAVSSplit], []uint64) {
+	stateChanges := make(types.StateTransitions[*OperatorAVSSplit])
+
+	stateChanges[0] = func(log *storage.TransactionLog) (*OperatorAVSSplit, error) {
+		operatorAVSSplit, err := oas.handleOperatorAVSSplitBipsSetEvent(log)
+		if err != nil {
+			return nil, err
+		}
+
+		slotId := base.NewSlotID(operatorAVSSplit.TransactionHash, operatorAVSSplit.LogIndex)
+
+		_, ok := oas.stateAccumulator[log.BlockNumber][slotId]
+		if ok {
+			fmt.Printf("Submissions: %+v\n", oas.stateAccumulator[log.BlockNumber])
```

Comment:
Just leaving a note here that we should remove this before merging to keep the logs tidy

Reply:
Oh was that just there for debugging?

Reply:
Do you want me to outright remove it or change the log description?

Reply:
Just remove any `fmt.Printf` calls you have